packages/agent/bin/helpers.ts

```diff
@@ -0,0 +1,31 @@
+import { main as effectionMain, Operation } from "effection";
+import { EventEmitter } from "events";
+
+export const self: Operation = ({ resume, context: { parent }}) => resume(parent);
+export const parent: Operation = ({ resume, context: { parent }}) => resume(parent.parent);
+
+export function main(operation: Operation) {
```

Comment:
Once we make `effection` into a monorepo, this function will live in `@effection/node`